packs/feats/mediums-awareness.json

```diff
@@ -24,7 +24,17 @@
             "remaster": true,
             "title": "Pathfinder War of Immortals"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "action:seek"
```

Comment:
Will this work for the initiative roll?

Reply:
No, I just missed it. I'm heading out for the night already but